_data/icons/ultron.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmS4W4kY7XYBA4f52vuuytXh3YaTcNBXF14V9tEY6SNqhz",
```

Comment:
where did you pin this?

Reply:
I looked, apparently the last time for some reason did not pass the pin.
Now I made a pin on chainsafe and infura
https://ultron.infura-ipfs.io/ipfs/QmS4W4kY7XYBA4f52vuuytXh3YaTcNBXF14V9tEY6SNqhz